features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,22 @@
+Given("I am on the {string} Page") do |page_name|
+  visit page_path(page_name)
+end
+
+Then("I fill in {string} with {string}") do |element, content|
+  fill_in(element, :with => content)
+end
+
+Then("I click on {string}") do |button_name|
+  click_link_or_button(button_name)
+end
+
+Then("I should see {string}") do |message|
+  page.should have_content message
+end
+
+def page_path(path)
+  site_path =  if path == 'Create Article'
+                new_article_path
+               end
+  site_path
```

Comment:
???

Reply:
Raoul showed this for us. It's a function that lets us reuse steps more the one time. In this case this is all it needs to do. But when we will test other paths will add then to the if statment.
 
If you want a real answers from now on you need to come with something better then  ``` ??? ``` .

Reply:
@diraulo Is this your implementation?

Reply:
@holgertidemand For future reference, you should learn to own the code you produce. The moment you get a code from a coach or someone else it becomes yours and you need to be able to defend your decision to use it without pointing at someone else. 

Here's what I showed you back then https://github.com/CraftAcademy/evensteven/pull/10#discussion_r171255271

Your team ended up going with the above implementation or something similar which I let through for various reasons related to the context in which you were working on. Now that you know better, this was not going to get an approval from me either.

Bottom line here is *own the code you produce* - You will not tell your boss tomorrow in a code review _I got this from StackOverflow, and I just used it..._ without thinking about its impact on the application.

Reply:
Copy that! @diraulo 
I will OWN me code! :D